assets/sass/patterns/organisms/listing.scss

```diff
@@ -34,6 +34,21 @@ $totalHighlightsSpacing: 6;
   .content-header--read-more {
     border: none;
   }
+
+  // When wide, hide to avoid duplication: then, the contents is also displayed in the right hand article column
+  .listing-list__item--related:first-child {
```

Comment:
I'm happy to remove `:first-child`, as it's the duplication we want to avoid, and doing so will not tie the matching of this selector to the related item's list position.